console/src/components/Services/Data/TableModify/ColumnEditorList.js

```diff
@@ -104,10 +104,29 @@ const ColumnEditorList = ({
 
       const isOk = getConfirmation(confirmMessage, true, colName);
       if (isOk) {
+        const pk = [];
+        let flag = true;
```

Comment:
You could perhaps change the name of this variable. Maybe something like `pkPresentFlag`? Also, adding a comment in the above line which would help others know why that flag variable is necessary would be awesome!

Reply:
Yeah. Will do the changes.